client/src/component/wingit/WingIt.jsx

```diff
@@ -86,9 +86,14 @@ class WingIt extends Component {
   listPlayers = (players) => {
     const playerList = players.map(function (player) {
       return (
-        <li key={player.id} className="my-player-list-item">
-          <h2>{player.name}</h2>
-        </li>
+        <div className="player-card-container">
```

Comment:
@Aidanchase This could be a candidate for a separate component later on if we have time to refactor